backends/ebpf/runtime/ebpf_runtime.c

```diff
@@ -33,13 +33,27 @@ void usage(char *name) {
            " into a filter function, and returns the output.\n" , name);
 }
 
+int pcap_while_loop(pcap_t *in_handle, pcap_dumper_t *out_handle) {
```

Comment:
The name of this function is not very good. It should be called something like "process_packets".